### ISSUE TITLE
E2E river implementation of ERC1155 gating

### ISSUE THREAD

**Issue by clemire** [river-build/river#558]
### Description

After #550 , add support for evaluating 1155s in typescript and go and check with e2e tests.

**Comment by texuf** [comment]
node implementation is being merged into the client, client effort is underway

**Comment by clemire** [comment]
client implementation in review: https://github.com/river-build/river/pull/964/files
